cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -372,11 +372,7 @@ spec:
                   type: integer
                   minimum: 1
                   maximum: 65535
-                securePort:
```

Comment:
Is there a reason get rid of this check in the CRD? It still seems valuable to users as a fail-fast check against errors, even if we are also double checking in the code.

Reply:
The problem is that `nullable` isn't supported on k8s 1.13 or earlier. The only alternative I can think of is to fork the common.yaml to have a different version for 1.14 or newer. This doesn't seem worth it, so we discussed just removing this check until 1.13 is no longer supported, and add the code check.